### DISCUSSION TITLE
Dynamic chart heading in sync of filter values

### DISCUSSION THREAD

**Discussion by Priya359** [apache/superset#26097] [Ideas]
Please provide feature to add filter selected value in chart heading in superset dashboard 
e.g : If there is month filter and i have selected August month, so can i show august value in chart heading or description in any way

**Comment by dsKanishk** [comment]
Why is it not a feature to begin with.
Any updates if there is a way to achieve this.
May be using jinja templating or something like that.
Although the feature is much awaited and feels like a necessary add on.
The filter box was a good feature in that sense.

**Comment by dsKanishk** [comment]
Try n explore Jinja Templating!
Hope that helps.
